test/images/nonewprivs/nnp.c

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Kubernetes Authors.
```

Comment:
Yes. For multi arch test images we are planning to have all images under `kubernetes/tests/images` directory for supported architectures like `amd64, arm, arm64, ppc64le`. So its always very easy to cross build the go code over a C code.